### PR TITLE
Migrate the Code Lens status icon to Octicons

### LINKED ISSUE #845: Use octicon to replace emojis
This is to request replacing the emojis in the editor view:
image

VS Code now supports oct icons, and by adopting them, the test status looks more consistent and appealing in the editor window.
https://code.visualstudio.com/api/references/icons-in-labels
https://octicons.github.com/

### PR DESCRIPTION
**Pull request by jdneo**
Fix #845.

Appearance:
image

image

### CODE REVIEW

**Comment by akaroml** [comment]
Nice. How about the cases that show a question mark?

File: src/codelens/TestCodeLensProvider.ts [hunk 1]
```diff
@@ -139,7 +138,7 @@ export class TestCodeLensProvider implements CodeLensProvider {
             return '?';
```
Comment (akaroml) [inline src/codelens/TestCodeLensProvider.ts]: Does vscode has an icon to represent the unknown state?

Comment (jdneo) [reply src/codelens/TestCodeLensProvider.ts]: A related one might be:

image

But looks too complicated.

Or maybe we can just show nothing if the state is unknown.

Comment (akaroml) [reply src/codelens/TestCodeLensProvider.ts]: $(question)

Comment (jdneo) [reply src/codelens/TestCodeLensProvider.ts]: Per offline discussion. We will leave the question icon as what it is.

**Comment by jdneo** [comment]
Result in VS Code 1.40.0 stable

image

**Pull request merged**